### PR TITLE
update GitHub IdP doc

### PR DESCRIPTION
**Pull request by alexfornuto**
## Summary

Updates the GitHub IdP doc for style and format.

## Related issues

https://github.com/pomerium/docs/issues/24

## Checklist

- [ ] reference any related issues
- [ ] updated docs
- [ ] add appropriate tag (`improvement` / `bug` / etc)
- [ ] ready for review

### CODE REVIEW

File: docs/docs/identity-providers/github.md [hunk 1]
```diff
@@ -9,63 +9,81 @@ meta:
...
-The format of the `idp_service_account` for GitHub is a base64-encoded JSON document:
+1. The format of the `idp_service_account` for GitHub is a base64-encoded JSON document:
 
-```json
-{
-  "username": "YOUR_GITHUB_USERNAME",
-  "personal_access_token": "GENERATED_GITHUB_ACCESS_TOKEN"
-}
-```
+   ```json
```
Comment (desimone) [inline docs/docs/identity-providers/github.md:52]:
```suggestion
   ```json tmp.json
```

maybe we should make the connection between this file, and the thing we are base64'ing explicit here (and in the other docs?)

File: docs/docs/identity-providers/github.md [hunk 2]
```diff
@@ -9,63 +9,81 @@ meta:
...
+```bash
+IDP_PROVIDER="github"
+IDP_CLIENT_ID="REDACTED"       // github application ID
+IDP_CLIENT_SECRET="REDACTED"   // github application secret
+IDP_SERVICE_ACCOUNT="REDACTED" // github service account (personal access token)
+```
+:::
+::::
 
 Whenever a user tries to access  your application integrated with Pomerium, they will be presented with a sign-on page as below:
```
Comment (desimone) [inline docs/docs/identity-providers/github.md:88]:
```suggestion
Whenever a user tries to access your application integrated with Pomerium, they will be presented with a sign-on page as below:
```

**Review by desimone — APPROVED** [review]

File: docs/docs/identity-providers/github.md (hunk 1 above)
Comment (alexfornuto) [reply docs/docs/identity-providers/github.md:52]: It doesn't appear that adding this changes anything in the render.

Comment (desimone) [reply docs/docs/identity-providers/github.md:52]: Ah, okay. I guess vuepress doesn't support file name rendering.

**Pull request merged**